kernel/kata_config_version

```diff
@@ -1 +1 @@
-26
+27
```

Comment:
Do you need this bump - you didn't change the kernel version or config file for instance, so the kernel we are building is the same...?

Reply:
Yeah, but the CI script asks for a bump when the `kernel/` directory gets modified. Maybe we should improve this on a following PR.

Reply:
ah, OK, fair enough. thx